workflow/snakemake_rules/export_for_nextstrain.smk

```diff
@@ -367,16 +367,41 @@ except:
     # means that the Snakefile won't crash.
     deploy_origin = "by an unknown identity"
 
+
+rule deploy_single:
+    input:
+        # Replication of all_regions but with build_name as wildcard
+        auspice_json = expand("auspice/{prefix}_{{build_name}}.json", prefix=config["auspice_json_prefix"]),
+        tip_frequencies_json = expand("auspice/{prefix}_{{build_name}}_tip-frequencies.json", prefix=config["auspice_json_prefix"]),
+        root_sequence_json = expand("auspice/{prefix}_{{build_name}}_root-sequence.json", prefix=config["auspice_json_prefix"]),
+        dated_auspice_json = expand("auspice/{prefix}_{{build_name}}_{date}.json", prefix=config["auspice_json_prefix"], date=config.get("build_date", get_todays_date())),
+        dated_tip_frequencies_json = expand("auspice/{prefix}_{{build_name}}_{date}_tip-frequencies.json", prefix=config["auspice_json_prefix"], date=config.get("build_date", get_todays_date())),
+        dated_root_sequence_json = expand("auspice/{prefix}_{{build_name}}_{date}_root-sequence.json", prefix=config["auspice_json_prefix"], date=config.get("build_date", get_todays_date()))
+    output:
+        temp(touch("auspice/{build_name}.deployed"))
+    params:
+        deploy_url = config["deploy_url"],
+        prefix = config["auspice_json_prefix"]
+    benchmark:
+        "benchmarks/deploy_single_{build_name}.txt"
+    run:
+        shell("nextstrain deploy {params.deploy_url:q} {input:q}")
+        if params.deploy_url == "s3://nextstrain-data":
+            slack_message = f"Deployed {wildcards.build_name} to https://nextstrain.org/{(params.prefix + '_' + wildcards.build_name).replace('_', '/')}"
+        else:
+            slack_message = f"Deployed {wildcards.build_name} to {params.deploy_url}"
+        send_slack_message(slack_message)
+
 rule deploy:

```

Comment:
This would be more clear to me:

```suggestion
rule deploy_all:
```

Reply:
It would be clearer but it would be a breaking change - as we explicitly request the `deploy` rule in some Github actions.

So rather than `deploy_all`/`deploy_single` it's less breaking to call them `deploy`/`deploy_single`